runtime/oti/j9nonbuilder.h

```diff
@@ -5760,6 +5761,9 @@ typedef struct JFRState {
 	int64_t prevProcTimestamp;
 	int64_t prevContextSwitchTimestamp;
 	uint64_t prevContextSwitches;
+	J9HashTable *typeIDs;
+	omrthread_monitor_t typeIDMonitor;
+	UDATA typeIDcount;
```

Comment:
This is likely queryable from the hash table itself (and won't introduce any inconsistency in the sizes).

Reply:
Comment no longer applies after tables moved to class loader.